airflow/providers/neo4j/operators/neo4j_operator.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
Again, here I think this should be ` airflow/providers/neo4j/operators/neo4j.py` now.